Some output being dropped since upgrading to `through2`

This was originally reported by @josemarluedke when he noticed TypeScript errors not being emitted, but after a little more investigation it looks like other output is missing as well.

In [@josemarluedke's reproduction repo](https://github.com/josemarluedke/ts-no-errors), running `ember build` only emits the following:

```
WARNING: ts-no-errors has added the class-properties plugin to its build, but ember-cli-babel provides these by default now! You can remove the transforms, or the addon that provided them, such as @ember-decorators/babel-transforms.
```

I can also briefly see the `Building...` spinner and `cleaning up` message, but those are quickly overwritten.

If I set a resolution for `console-ui` to `3.0.0` and rerun `yarn`, I see the full output I'd expect:

```
WARNING: ts-no-errors has added the class-properties plugin to its build, but ember-cli-babel provides these by default now! You can remove the transforms, or the addon that provided them, such as @ember-decorators/babel-transforms.
WARNING: ts-no-errors has added the class-properties plugin to its build, but ember-cli-babel provides these by default now! You can remove the transforms, or the addon that provided them, such as @ember-decorators/babel-transforms.
Environment: development
app/components/navigation.ts:4:3 - error TS2564: Property 'someArg' has no initializer and is not definitely assigned in the constructor.

4   someArg: boolean;
    ~~~~~~~

app/components/navigation.ts:6:7 - error TS2378: A 'get' accessor must return a value.

6   get shouldGiveAnError() {
        ~~~~~~~~~~~~~~~~~

app/components/navigation.ts:7:5 - error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'Boolean' has no compatible call signatures.

7     this.someArg();
      ~~~~~~~~~~~~~~

tests/integration/components/navigation-test.ts:15:18 - error TS2531: Object is possibly 'null'.

15     assert.equal(this.element.textContent.trim(), '');
                    ~~~~~~~~~~~~~~~~~~~~~~~~

tests/integration/components/navigation-test.ts:24:18 - error TS2531: Object is possibly 'null'.

24     assert.equal(this.element.textContent.trim(), 'template block text');
                    ~~~~~~~~~~~~~~~~~~~~~~~~

cleaning up...
Built project successfully. Stored in "dist/".
```

--- a/lib/index.js
+++ b/lib/index.js
@@ -36,9 +36,10 @@ class UI {
     let options = _options || {};
     // Output stream
     this.actualOutputStream = options.outputStream || process.stdout;
-    this.outputStream = through(function (data) {
+    this.outputStream = through(function (data, enc, callback) {
       spinner.stop();
-      this.emit('data', data);
+
+      callback(null, data);
     });
     this.outputStream.setMaxListeners(0);
     this.outputStream.pipe(this.actualOutputStream);

